packages/common/src/extensions/slickCellExcelCopyManager.ts

```diff
@@ -142,13 +142,16 @@ export class SlickCellExcelCopyManager {
       clipboardCommandHandler: (editCommand: EditCommand) => {
         this._undoRedoBuffer.queueAndExecuteCommand.call(this._undoRedoBuffer, editCommand);
       },
-      dataItemColumnValueExtractor: (item: any, columnDef: Column) => {
+      dataItemColumnValueExtractor: (item: any, columnDef: Column, row: number = 0, cell: number = 0) => {
```

Comment:
you can omit `number` type on both arguments since providing `0` is inferring a type `number`